internal/cmd/asyncapi/command_export.go

```diff
@@ -575,3 +588,21 @@ func createConsumer(broker string, clusterCreds *v1.APIKeyPair, groupId string)
 	}
 	return consumer, nil
 }
+
+// Check if topic matches user-specified topics/prefixes. True if user didn't specify
+func topicMatch(topic string, topics types.Set, prefixes []string) bool {
```

Comment:
We can keep one set/slice for all of our topics / prefixed topics, and check for the trailing "*" in this function.